### DISCUSSION TITLE
Allow to display dropdown content directly when clicking on the hamburger icon in mobile view

### DISCUSSION THREAD

**Discussion by ansavicus** [elementor/elementor#22653] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It is very difficult to adapt the responsive view of megamenu to mobile devices. It would be nice to be able to hide some menu items in mobile view (even if they are used in desktop view) and above all, to be able to display the dropdown content of the megamenu directly when clicking on the hamburger menu icon.

### Describe the solution you'd like

For example, on my site I would like to display the content that appears inside the mega menu item called "Services" directly when the menu is pulled down in the mobile view.

![Screenshot_2](https://github.com/elementor/elementor/assets/104578458/8b2309df-9328-409f-9cf7-18c5f95cb3f8)

In short, hide the title "Servicios" and show its content directly when clicking on the menu icon. I've already managed to hide the sections I don't want to show using custom CSS, but I haven't managed to do this.

![Screenshot_2023-06-02-22-06-54-39_e4424258c8b8649f6e67d283a50a2cbc](https://github.com/elementor/elementor/assets/104578458/5838bcbd-895f-466b-b4b2-a123785710dc)
